internal/functions/deploy/bundle.go

```diff
@@ -146,9 +147,45 @@ func GetBindMounts(cwd, hostFuncDir, hostOutputDir, hostEntrypointPath, hostImpo
 		if err != nil {
 			return nil, err
 		}
-		modules := importMap.BindHostModules()
+
+		modules := utils.BindHostModules(importMap)
 		dockerImportMapPath := utils.ToDockerPath(hostImportMapPath)
 		modules = append(modules, hostImportMapPath+":"+dockerImportMapPath+":ro")
+
+		addFile := func(srcPath string, w io.Writer) error {
+			f, err := fsys.Open(filepath.FromSlash(srcPath))
+			if err != nil {
+				return errors.Errorf("failed to read file: %w", err)
+			}
+			defer f.Close()
+			if fi, err := f.Stat(); err != nil {
+				return errors.Errorf("failed to stat file: %w", err)
+			} else if fi.IsDir() {
+				return errors.New("file path is a directory: " + srcPath)
+			}
+
+			r := io.TeeReader(f, w)
+			_, err = io.Copy(io.Discard, r) // Discard the read data after writing to w
```

Comment:
```suggestion
			_, err = io.Copy(w, f) // Discard the read data after writing to w
```

we don't need to tee here, just copy from file to the writer will do.

Reply:
Done at [e857e1f](https://github.com/supabase/cli/pull/3417/commits/e857e1f42b015420e88cce4aea35c268617bc66b)